examples/basic/index.ts

```diff
@@ -0,0 +1,90 @@
+import { AppLoaderPlugin, Shadow } from 'pixi-shadows';
+import { Application, Container, InteractionEvent, SCALE_MODES, Sprite, Texture } from 'pixi.js';
+
+// Initialise the shadows plugin
+Application.registerPlugin(AppLoaderPlugin);
+/* The actual demo code: */
+
+// Create your application
+const width = 800;
+const height = 500;
+const app = new Application({ width, height, autoStart: true });
+
+document.body.appendChild(app.view);
+// AppLoaderPlugin.init.apply(app);
+
+// Create a world container
+const world = app.shadows.container;
+
+// A function to combine different assets of your world object, but give them a common transform by using pixi-layers
+// It is of course recommended to create a custom class for this, but this demo just shows the minimal steps required
+function createShadowSprite(texture: Texture, shadowTexture: Texture) {
+    const container = new Container(); // This represents your final 'sprite'
+
+    // Things that create shadows
+    if (shadowTexture) {
+        const shadowCastingSprite = new Sprite(shadowTexture);
+
+        shadowCastingSprite.parentGroup = app.shadows.casterGroup;
+        container.addChild(shadowCastingSprite);
+    }
+
+    // The things themselves (their texture)
+    const sprite = new Sprite(texture);
+
+    container.addChild(sprite);
+
+    return container;
+}
+
+// Can set ambientLight for the shadow filter, making the shadow less dark:
+// shadows.filter.ambientLight = 0.4;
+
+// Create a light that casts shadows
+const shadow = new Shadow(700, 1);
+
+shadow.position.set(450, 150);
+world.addChild(shadow);
+
+// Create a background (that doesn't cast shadows)
+const bgTexture = Texture.from('../../assets/background.jpg');
```

Comment:
```suggestion
const bgTexture = Texture.from(bgTexturePath);
```
In accordance with the previous suggestion